db/migrate/20170215123538_add_merge_queue_enabled_to_stacks.rb

```diff
@@ -0,0 +1,5 @@
+class AddMergeQueueEnabledToStacks < ActiveRecord::Migration[5.0]
```

Comment:
Since Rails 5.0, migrations are tagged with the Rails version against which they were wrote. So that in the future Rails can do backward incompatible changes to the migration API, without breaking historic migrations.